tests/unit/macaroons/test_services.py

```diff
@@ -197,3 +197,61 @@ def test_get_macaroon_by_description(self, macaroon_service):
             macaroon_service.get_macaroon_by_description(user.id, macaroon.description)
             == dm
         )
+
+    def test_check_if_macaroon_exists_unprefixed_macaroon(self, macaroon_service):
+        raw_macaroon = pymacaroons.Macaroon(
+            location="fake location",
+            identifier=str(uuid4()),
+            key=b"fake key",
+            version=pymacaroons.MACAROON_V2,
+        ).serialize()
+
+        with pytest.raises(services.InvalidMacaroon):
+            macaroon_service.check_if_macaroon_exists(raw_macaroon)
+
+    def test_check_if_macaroon_exists_no_macaroon(self, macaroon_service):
+        raw_macaroon = pymacaroons.Macaroon(
+            location="fake location",
+            identifier=str(uuid4()),
+            key=b"fake key",
+            version=pymacaroons.MACAROON_V2,
+        ).serialize()
```

Comment:
Maybe move the setup part into a fixture?
```suggestion
    @pytest.fixture
    def raw_macaroon()
        return pymacaroons.Macaroon(
            location="fake location",
            identifier=str(uuid4()),
            key=b"fake key",
            version=pymacaroons.MACAROON_V2,
        ).serialize()

    def test_check_if_macaroon_exists_unprefixed_macaroon(self, macaroon_service, raw_macaroon):
        with pytest.raises(services.InvalidMacaroon):
            macaroon_service.check_if_macaroon_exists(raw_macaroon)

    def test_check_if_macaroon_exists_no_macaroon(self, macaroon_service, raw_macaroon):
```